templates/multistage-buildconfig.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata: 
+  name: jlinked-image
+spec:
+  source:
+    dockerfile: |
+	#Stage-1: ubi9-jlinked-image is builder image + application + jlinked JVM
+	# install additional system dependencies (for ubi-micro) to /mnt/jrootfs
+	FROM ubi9-jlinked-image AS ubi9-jlinked-image  # Replace this with appropriate base image
```

Comment:
We wil want OpenShift to substitute `FROM ubi9-jlinked-image` with the correct imagestream name/URI for the specific instance of this template once deployed. To achieve that, change this line to `FROM - AS ubi9-jlinked-image`; and then we also need to add some more metadata elsewhere in the buildConfig, a `dockerStrategy` key underneath `strategy:`, which will describe the ImageStream to pull from. See https://docs.openshift.com/container-platform/4.15/cicd/builds/build-strategies.html#builds-strategy-docker-from-image_build-strategies

Reply:
Aha, quoting that URI, we may have a problem: "If the Dockerfile uses multi-stage builds, the image in the last FROM instruction will be replaced.", so in this case, that would be the ubi-micro below, not what we want.

We might have to switch from a mutistage dockerfile to two separate, linked OpenShift buildConfigs instead.